docs/connector-development/cdk-python/README.md

```diff
@@ -1,44 +1,55 @@
 # Connector Development Kit
 
 :::info
-Over the next few months, the project will only accept connector contributions that are made using the [Low-Code CDK](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview) or the [Connector Builder](https://docs.airbyte.com/connector-development/connector-builder-ui/overview).
+Over the next few months, the project will only accept connector contributions that are made using the
+[Low-Code CDK](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview) or the
+[Connector Builder](https://docs.airbyte.com/connector-development/connector-builder-ui/overview).
 
-Contributions made with the Python CDK will be closed, but we will inquire to understand why it wasn't done with Low-Code/Connector Builder so we can address missing features.
-This decision is aimed at improving maintenance and providing a larger catalog with high-quality connectors.
+New pull requests made with the Python CDK will be closed, but we will inquire to understand why it wasn't done with
```

Comment:
/cc @marcosmarxm, I've tried to clarify the wording here a little bit, but that is your domain! Let me know if I should roll any of this back.